hugegraph-python-client/README.md

```diff
@@ -17,7 +17,7 @@ release soon
 ```python
 from pyhugegraph.client import PyHugeClient
 
-client = PyHugeClient("127.0.0.1", "8080", user="admin", pwd="admin", graph="hugegraph")
+client = PyHugeClient("127.0.0.1", "8080", user="admin", pwd="admin", graph="hugegraph", gs="DEFAULT")
```

Comment:
- add a comment for this line (like `gs` is optional)
- currently `gs` compatible with `graphspace`? (set as alias)